backend-tests/tests/test_rbac_v2.py

```diff
@@ -1508,3 +1511,601 @@ def test_get_emails_by_group(self, clean_mongo, test_case):
+                },
+                "roles": [
+                    {
+                        "name": "test",
+                        "permission_sets_with_scope": [
+                            {
+                                "name": "ReadReleases",
+                                "scope": {"type": "ReleaseTags", "value": ["foo"],},
+                            },
+                            {
+                                "name": "ManageReleases",
+                                "scope": {"type": "ReleaseTags", "value": ["foo"],},
+                            },
+                            {"name": "DeployToDevices",},
+                        ],
+                    },
+                ],
+                "tags": ["foo", "bar"],
+            },
+        ],
+    )
+    def test_manage_releases(self, clean_mongo, test_case):
+        """
+        Tests access to releases with releases scope.
+        """
+        self.logger.info("RUN: %s", test_case["name"])
+
+        uuidv4 = str(uuid.uuid4())
+        tenant, username, password = (
+            "test.mender.io-" + uuidv4,
+            "admin+" + uuidv4 + "@example.com",
+            "secretsecret",
+        )
+        tenant = create_org(tenant, username, password, "enterprise")
+        create_roles(
+            tenant.users[0].token, test_case["roles"], status_code=201,
+        )
+        # create and upload artifacts
+        artifacts = (
+            {"artifact_name": "foo", "device_types": ["arm1"], "size": 256},
+            {"artifact_name": "bar", "device_types": ["arm1"], "size": 256},
+        )
+        api_client = ApiClient(deployments.URL_MGMT)
+        api_client_v2 = ApiClient(deployments_v2.URL_MGMT)
+        api_client.headers = {}  # avoid default Content-Type: application/json
+        api_client.with_auth(tenant.users[0].token)
+        for artifact_kw in artifacts:
+            with get_mender_artifact(**artifact_kw) as artifact:
+                r = api_client.call(
+                    "POST",
+                    deployments.URL_DEPLOYMENTS_ARTIFACTS,
+                    files=(
+                        ("description", (None, "description")),
+                        ("size", (None, str(os.path.getsize(artifact)))),
+                        (
+                            "artifact",
+                            (
+                                artifact,
+                                open(artifact, "rb"),
+                                "application/octet-stream",
+                            ),
+                        ),
+                    ),
+                )
+            assert r.status_code == 201
+
+        # get all releases
+        api_client_v2.with_auth(tenant.users[0].token)
+        r = api_client_v2.call("GET", deployments_v2.URL_RELEASES)
+        assert r.status_code == 200
+        all_releases = r.json()
+        assert len(all_releases) == 2
+
+        # switch to user with limited access
```

Comment:
ok just for me, and to double check which error code represents the limits?

Reply:
not sure I understand the question